packages/core/src/components/tooltip/tooltip.tsx

```diff
@@ -175,27 +186,42 @@ export class Tooltip {
     );
   }
 
+  private clearHideTimeout() {
+    if (this.interactive) {
+      clearTimeout(this.hideTooltipTimeout);
+    }
+  }
+
   private queryAnchorElements() {
     return Array.from(document.querySelectorAll(this.for));
   }
 
   private registerTriggerListener() {
     const elements = this.queryAnchorElements();
     elements.forEach((e) => {
-      e.addEventListener('mouseenter', this.onMouseEnterBind);
-      e.addEventListener('mouseleave', this.onMouseLeaveBind);
+      e.addEventListener('mouseenter', this.onEnterElementBind);
+      e.addEventListener('mouseleave', this.onLeaveElementBind);
+      e.addEventListener('focusin', this.onEnterElementBind);
+      e.addEventListener('focusout', this.onLeaveElementBind);
+      e.setAttribute('aria-describedby', 'ix-tooltip-' + this.id);
```

Comment:
Why do we want to do this? We should avoid influencing other components out of the component scope.

Reply:
Its an a11y requirement as specified as follows on the URL https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role:

**Associated WAI-ARIA roles, states, and properties**

The element that serves as the tooltip container has role="tooltip" set.
The element that triggers the tooltip references the tooltip element with aria-describedby.

**Keyboard interactions**
Escape
: Closes the tooltip

The tooltip should appear on focus or when the element is hovered on, without additional interaction. It should disappear automatically when the focus on the owning element is lost or the mouse is moved outside the owning element and the tooltip. While the tooltip does not receive focus, the Escape should close it if it is open.